src/definitions/modules/slider.js

```diff
@@ -267,12 +277,15 @@ $.fn.slider = function(parameters) {
           },
           touchEvents: function() {
             module.verbose('Binding touch events');
-            $module.find('.track, .thumb, .inner').on('touchstart' + eventNamespace, function(event) {
+            var downElements = settings.slideEverywhere ? '.thumb,.inner,.track' : '.thumb';
+            $module.find(downElements).on('touchstart' + eventNamespace, function(event) {
               event.stopImmediatePropagation();
               event.preventDefault();
               module.event.down(event);
             });
-            $module.on('touchstart' + eventNamespace, module.event.down);
+            if(!settings.slideEverywhere) {
+              $module.on('touchstart' + eventNamespace, module.event.down);
+            }
```

Comment:
This should be the same logic as you did for mouse events, shouldn't it?

Reply:
Fixed in b48c4f69718c8f777df8f4a36dc89a9b211fc3a9.

I have mistakenly assumed that both functions (`touchEvents` and `mouseEvents`) are called on touch devices. In addition, I got the logic meaning of `settings.slideEverywhere` wrong in that if statement.